scene/2d/tile_map_layer.h

```diff
@@ -31,6 +31,7 @@
 #ifndef TILE_MAP_LAYER_H
 #define TILE_MAP_LAYER_H
 
+#include "core/os/mutex.h"
```

Comment:
```suggestion
```
Already included

Reply:
Only indirectly. I guess the "requires" in https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html#:~:text=Godot%20and%20third%2Dparty%20headers%20should%20be%20included%20in%20the%20file%20that%20requires%20them%2C%20i.e.%20in%20the%20.h%20header%20if%20used%20in%20the%20declarative%20code%20or%20in%20the%20.cpp%20if%20used%20only%20in%20the%20imperative%20code. really means it, and you don't include-what-you-use?

Done

Reply:
Yep, we avoid it as it clutters a lot